warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/TableSplitsCache.java

```diff
@@ -141,18 +193,23 @@ public void writeCacheFile(FileSystem fs, Path tmpSplitsFile) throws IOException
         }
         Set<String> tableNames = getIngestTableNames();
         Map<String,Integer> splitsPerTable = new HashMap<>();
+        Map<String,Map<Text,String>> tmpSplitLocations = new HashMap<>();
         if (tops != null) {
             try (PrintStream out = new PrintStream(new BufferedOutputStream(fs.create(tmpSplitsFile)))) {
                 this.splits = new HashMap<>();
                 // gather the splits and write to PrintStream
                 for (String table : tableNames) {
                     log.info("Retrieving splits for " + table);
+                    Map<Text,String> splitLocations = getSplitsWithLocation(table);
```

Comment:
If we already have all of the split locations, why get the splits again in the next line?

Reply:
oversight.  good catch